data/ui_config.json

```diff
@@ -0,0 +1,77 @@
+[
```

Comment:
Should we include this config in core?

Reply:
seems like it needs some default value, will update it